packages/maskbook/src/social-network-adaptor/twitter.com/encoding.ts

```diff
@@ -68,7 +68,7 @@ export const twitterEncoding = {
             return [
                 `🎼${batchReplace(
                     payload
-                        .replace(/^PostData_v\d=/i, '')
+                        .replace(/^postdata_v\d=/i, '')
```

Comment:
I don't think this change is good. The uppercase makes it easier to read.

Reply:
This one didn't get resolved. This rule seems decrease the readability of the code.

Reply:
https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/better-regex.md

Reply:
This link doesn't help with this problem. `PostData` is more readable than `postdata`